packages/core/src/interactable-card/InteractableCard.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  forwardRef,
+  MouseEvent,
+  ComponentPropsWithoutRef,
+  SyntheticEvent,
+  useRef,
+} from "react";
+import { clsx } from "clsx";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { capitalize, makePrefixer, useControlled, useForkRef } from "../utils";
+import { useInteractableCard } from "./useInteractableCard";
+import interactableCardCss from "./InteractableCard.css";
+import {
+  InteractableCardValue,
+  useInteractableCardGroup,
+} from "./InteractableCardGroupContext";
+
+const withBaseName = makePrefixer("saltInteractableCard");
+
+export interface InteractableCardProps extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * Accent border position: defaults to "bottom"
+   */
+  accent?: "bottom" | "top" | "left" | "right";
+  /**
+   * **Deprecated:** Use the `accent` prop instead
+   *
+   */
+  accentPlacement?: "bottom" | "top" | "left" | "right";
+  /**
+   * If `true`, the card will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * If `true`, the card will have selected styling.
+   */
+  selected?: boolean;
```

Comment:
Is it intentional that there's on a controlled API?

Reply:
Yeah I'm not sure about when this would be used but it's a state in the design.

Reply:
I am going to remove this prop and we can add it back later alongside an uncontrolled version if necessary